packages/evolution-generator/src/scripts/generate_choices.py

```
@@ -5,29 +5,95 @@
     is_excel_file,
     is_ts_file,
     get_workbook,
     sheet_exists,
     get_headers,
 )
-from scripts.labels_generator import LabelFormatter
+from scripts.labels_generator import LabelFormatter, LabelsGenerator
 
 
-def process_label(text):
+def _process_label_yaml(text):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_process_label_yaml`

(ANN202)